.github/workflows/test.yml

```diff
@@ -167,15 +167,30 @@ jobs:
           arch: x86_64
           action: test
           cache: true
-        - name: 'WAMR on Linux/x86_64'
-          engine: 'wamr'
+        - name: 'WAMR interp on Linux/x86_64'
+          engine: 'wamr-interp'
           repo: 'com_github_bytecodealliance_wasm_micro_runtime'
           os: ubuntu-20.04
           arch: x86_64
           action: test
           flags: --config=clang
-        - name: 'WAMR on macOS/x86_64'
-          engine: 'wamr'
+          deps: lld-12
+        - name: 'WAMR jit on Linux/x86_64'
+          engine: 'wamr-jit'
+          repo: 'com_github_bytecodealliance_wasm_micro_runtime'
+          os: ubuntu-20.04
+          arch: x86_64
+          action: test
+          flags: --config=clang
+          deps: lld-12
+        - name: 'WAMR interp on macOS/x86_64'
+          engine: 'wamr-interp'
+          repo: 'com_github_bytecodealliance_wasm_micro_runtime'
+          os: macos-11
+          arch: x86_64
+          action: test
```

Comment:
Style: consider ordering this: `interp on Linux`, `interp on macOS`, `jit on Linux`, `jit on macOS`.